### ISSUE TITLE
DMs are missing from profile pages in the iOS app

### ISSUE THREAD

**Issue by FediVideos** [mastodon/mastodon-ios#374]
I just heard from a new user confused about how to DM people in the official app.

There is currently no option for "direct message" on profiles viewed through the iOS app. (There is a reply button at the top but this defaults to public, so this is possibly making things more confusing.)

Could there be a DM icon on every profile page, maybe represented by an envelope, which creates a new direct message to the owner of the profile? I think this is the kind of feature that new users would be expecting and looking for on a social media platform.

**Comment by maciboy** [comment]
https://mastodon.social/@janboehm/108206034756979645

Here Jan Böhmermann asked the same question to Eugen…

**Comment by RobATX** [comment]
I agree. We need a direct message button somewhere as well on the profile page. On your own profile page we have hashtag, bookmark, and favorite but not direct message.
